client/web/web.go

```diff
@@ -57,20 +58,28 @@ type Server struct {
 	devMode  bool
 	devProxy *httputil.ReverseProxy // only filled when devMode is on
 
+	cgiMode    bool
 	apiHandler http.Handler // csrf-protected api handler
 }
 
+type ServerOpts struct {
+	DevMode     bool
+	CGIMode     bool
+	LocalClient *tailscale.LocalClient
+}
+
 // NewServer constructs a new Tailscale web client server.
 //
 // lc is an optional parameter. When not filled, NewServer
```

Comment:
Maybe move this doc to be on the `LocalClient` field in `ServerOpts`

Reply:
(specifically the "lc is an optional..." part)

Reply:
done